components/molecules/ProfileLanguageChart/profile-language-chart.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+
+import { AllSimpleColors, LanguageObject } from "../CardHorizontalBarChart/card-horizontal-bar-chart";
+
+import colors from "../../../lib/utils/color.json";
+
+const languageToColor: AllSimpleColors = colors as AllSimpleColors;
+const NOTSUPPORTED = "#64748B";
+interface ProfileLanguageChartProps {
+  languageList: LanguageObject[];
+}
+const ProfileLanguageChart = ({ languageList }: ProfileLanguageChartProps) => {
+  const sortedLangArray = languageList.sort((a, b) => b.percentageUsed - a.percentageUsed);
```

Comment:
Create a copy of the original array before mutating it with `sort`

```suggestion
  const sortedLangArray = languageList.slice().sort((a, b) => b.percentageUsed - a.percentageUsed);
```